src/SDKs/Storage/Management.Storage/Generated/Models/Action.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Storage.Models
+{
+    using Azure;
+    using Management;
+    using Storage;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System.Runtime;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines values for Action.
+    /// </summary>
+    [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public enum Action
```

Comment:
@JasonYang-MSFT Looking at the swagger, you have provided the correct information.  My concern is that this property is not required, so that a user doesn't have to provide iot.  What happens if this property is not provided by the user?  I think you need to explicitly test this.